service_discovery_manager/Component/mmDiscovery/monitor_server.cpp

```diff
@@ -32,6 +31,7 @@
 #endif
 
 #include "monitor_server.h"
+#include "ifaddrs.h"
```

Comment:
Can it work on Windows?

Reply:
Tizen build has been failed with this patch, please check it.

Reply:
move under !WIN32.

Reply:
I will check tizen build and update result

Reply:
I've updated on latest patch-set.
android and tizen build works fine.